Controllers/Article.controller.js

```diff
@@ -1,24 +1,52 @@
-const selectArticle = require("../Models/Article.model.js");
+const { selectArticles, updateVotes } = require("../Models/Article.model.js");
 
-const getArticle = (req, res, next) => {
-  selectArticle(req.params.article_id)
-    .then((article) => {
+const letsUpdateVotes = (req, res) => {
+  const article_id = req.params.article_id;
+  const additionalVotes = req.body.inc_votes;
+  if (!isNaN(article_id) && !isNaN(additionalVotes)) {
+    updateVotes(article_id, additionalVotes).then((article) => {
       if (article) {
-        res
-        .status(200)
-        .send({article:article});
+        res.status(200).send({ article: article });
       } else {
-        res
-          .status(404)
-          .send({
-            error: `No article was found with id: ${req.params.article_id}`,
-          });
+        res.status(404).send({
+          error: `No article was found with id: ${req.params.article_id}`,
+        });
       }
-    })
-    .catch((error) => {
-      next(error);
     });
+  } else if (isNaN(article_id)) {
+    res.status(400).send({
+      error: `article id must be a number`,
+    });
+  } else {
+    res.status(400).send({
+      error: `inc_votes must be a number`,
+    });
+  }
 };
 
+const getArticles = (req, res, next) => {
+  const article_id = req.params.article_id;
+  if (!isNaN(article_id)){
```

Comment:
We shouldn't be doing all of our error handling in the controllers. This will lead to a lot of repetition. We should have error handlers in app that catch our errors and deal with them appropriately.

Reply:
ah yea, i will address this in a new pr. Need to think of the best way of doing this. or if you have any best practices suggestions?
thinking of something like this:

const checkIsANumber(field_name, getField)(req, res, next) {
  if(isNaN(getField(req))){
    res.status(400).send(`blah blah ${field_name} blah blah`)
  } else {
    next(req, res)
  }
}